containers/datalab/Dockerfile.in

```diff
@@ -128,15 +128,19 @@ RUN ipython profile create default && \
     jupyter notebook --generate-config && \
     cd /datalab/lib/datalab-0.1.* && pip install . && \
     jupyter nbextension install --py datalab.notebook && \
-    cd /datalab/web && /tools/node/bin/npm install && \
-    cd / && \
-    ln -s /datalab/web/static/extensions/charting.js /datalab/nbconvert/charting.js && \
-    ln -s /datalab/web/static/require/element.js /datalab/nbconvert/element.js && \
-    ln -s /datalab/web/static/require/style.js /datalab/nbconvert/style.js && \
-    ln -s /datalab/web/static/require/visualization.js /datalab/nbconvert/visualization.js && \
+    ln -s /usr/local/share/jupyter/nbextensions/gcpdatalab/charting.css /datalab/nbconvert/charting.css && \
```

Comment:
Make a copy of them so we don't have to add a new entry every time we add a new file?